nntrainer/tensor/tensor.cpp

```diff
@@ -978,6 +978,68 @@ Tensor Tensor::dot(Tensor const &m, bool trans, bool trans_m) const {
 
   return output;
 }
+/**
+ * @brief compute the derivative of this in the current tensor
+ * @todo will have to see if beta effects this computation
+ */
+Tensor &Tensor::dot_deriv_wrt_1(Tensor const &m, Tensor const &output_deriv,
```

Comment:
following naming scheme of camel case? (and for others in this commit)

Reply:
Over time, the naming scheme has mixed up camel case (strided, and now this).
I will add an issue to fix the naming scheme for all the cases which is out of scope for this PR.